packages/turbo-r2-archive/src/autoCacheBust.ts

```diff
@@ -0,0 +1,17 @@
+async function bustEntireCache(list: R2Objects, env: Env) {
+	for (const object of list.objects) {
+		await env.R2_ARTIFACT_ARCHIVE.delete(object.key);
+	}
+}
+
+export async function bustOldCache(env: Env, cursor?: string) {
```

Comment:
Could this take `env.R2_ARTIFACT_ARCHIVE` instead of `env`, so it's not specific to a binding name?